relay-server/src/actors/processor.rs

```diff
@@ -2262,6 +2279,7 @@ impl EnvelopeProcessorService {
             if_processing!({
                 self.process_unreal(state)?;
                 self.create_placeholders(state);
+                self.enrich_event_with_dsc(state);
```

Comment:
wasn't sure where to put this logic, lmk if approach makes sense or not.

Reply:
I would rather put it in store normalization, like we do for the trace context:

https://github.com/getsentry/relay/blob/8afa67264a69d5d9105264bf94254aa0bfb10dd8/relay-general/src/store/normalize.rs#L195-L201

which pulls information from the DSC here:

https://github.com/getsentry/relay/blob/8afa67264a69d5d9105264bf94254aa0bfb10dd8/relay-server/src/actors/processor.rs#L1892

Reply:
makes sense!